src/components/landing/landing.css

```
@@ -0,0 +1,483 @@
+  padding: 6px 10px;
+  border: 1px solid hsl(var(--primary) / 0.35);
+  border-radius: 999px;
+  background: hsl(var(--primary) / 0.08);
+}
+.lp-beta-dot {
+  width: 7px; height: 7px; border-radius: 50%;
+  background: hsl(var(--primary));
+  box-shadow: 0 0 0 0 hsl(var(--primary) / 0.6);
+  animation: lpBetaPulse 1.8s ease-out infinite;
+}
+@keyframes lpBetaPulse {
```

Stylelint (17.7.0)

[error] Expected keyframe name "lpBetaPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)